src/controllers/artist.js

```diff
@@ -0,0 +1,121 @@
+// const { restart } = require('nodemon');
```

Comment:
There is a LOT of dead code in here! Once you have a working solution the old work in progress code should be removed as it doesnt add anything. If you discovered something in the process of working out your solution that should be conveyed to a future dev or reviewer (for example a quirk of the system that is out of scope of the changes you are making that was not immediately obvious) you could include it in a comment to help them out with an explanation. 
This is the BEST use of comments, something that future you will thank past you for including and make things easier for everyone to understand

Reply:
Very good point- I can see why the dead code and comments don't really add anything and can be very confusing.